extensions/amp-analytics/0.1/vendors.js

```diff
@@ -253,6 +253,24 @@ export const ANALYTICS_CONFIG = {
     }
   },
 
+  'piano': {
+    'requests': {
+      'host': 'https://api-v3.tinypass.com',
+      'basePrefix': '/api/v3',
+      'baseSuffix': '&pageview_id=${pageViewId}&rand=${random}',
+      'pageview': '${host}${basePrefix}/page/track?url=${canonicalUrl}&' +
+      'referer=${documentReferrer}&content_created=${contentCreated}&' +
+      'content_author=${contentAuthor}&content_section=${contentSection}&' +
+      'timezone_offset=${timezone}&tags=${tags}&amp_url=${ampdocUrl}&' +
+      'screen=${screenWidth}x${screenHeight}',
+    },
+    'transport': {
```

Comment:
OK will do. As per the previous code review, after I remove the `transport` block, should I keep the trailing comma at the end of the `requests`?